tooling/bundler/src/bundle/windows/templates/installer.nsi

```diff
@@ -679,6 +679,9 @@ Section Uninstall
   {{/each}}
   RMDir "$INSTDIR"
 
+  WinShell::UninstAppUserModelId "${BUNDLEID}"
+  WinShell::UninstShortcut "$DESKTOP\${MAINBINARYNAME}.lnk"
```

Comment:
We already remove the desktop shortcut on Line 691

We shouldn't remove the shortcut on uninstall, otherwise when updating it will be removed and never re-created again. We should detect if we are updating and skip removing the shortcut.

Reply:
This is what they did in their example, I don't know if we should do it or not, I can take a look tomorrow

I don't think uninstall runs on update, right?

Reply:
> We already remove the desktop shortcut on Line 691

Oh, was on mobile, forgot that this (`UninstShortcut`) is to unpin the shortcut